dev/com.ibm.ws.jdbc/src/com/ibm/ws/jdbc/internal/JDBCDriverService.java

```diff
@@ -719,10 +719,26 @@ public ClassLoader getClassLoaderForLibraryRef() {
      * @return Driver instance that accepts the URL. NULL if no such Driver can be loaded.
      * @throws SQLException if an error occurs.
      */
-    private Driver loadDriver(String url, ClassLoader classloader) throws SQLException {
+    private Driver loadDriver(String url, ClassLoader classloader, Properties props, String dataSourceID) throws SQLException {
         final boolean trace = TraceComponent.isAnyTracingEnabled();
         if (trace && tc.isDebugEnabled())
             Tr.entry(this, tc, "loadDriver", url, classloader);
+        int index = url.toLowerCase().indexOf("logintimeout");
+        if(index != -1) {
+            int length = url.length();
+            index += 13;  //length of logintimeout=
+            if(index+1 < length && !url.substring(index, index+1).equals("0")) {
```

Comment:
Is it worth it to check that the subsequent character is non-numeric to rule out the possibility of loginTimeout=060&...
?